lib/BlockedBy.js

```diff
@@ -154,6 +173,13 @@ class BlockedBy extends NodeResque.Plugin {
         // enqueueIn uses milliseconds
         await this.queueObject.enqueueIn(
             this.reenqueueWaitTime() * 1000 * 60, this.queue, this.func, this.args);
+
+        await helper.updateBuildStatus({
+            redisInstance: this.queueObject.connection.redis,
+            buildId,
+            status: 'BLOCKED',
+            statusMessage: `Blocked by these running jobs: ${blockingJobsRunning}`
```

Comment:
Can we do the buildIds instead? It's easier to look up in the UI

Reply:
Sure I can do that